### DISCUSSION TITLE
GPU acceleration, how?

### DISCUSSION THREAD

**Discussion by Sizzler** [janhq/jan#1523] [General]
Installed Jan and the nVidia Cuda 12.3 framework and in a video i watched about setting up Jan, the guys shows in settings>>advanced that he enables GPU acceleration there, but I dont have that option, nor do I actually see any options there except experimental and "Open the directory where the app data is located."

How do you enable GPU acc? my GPU is an RTX3070

**Comment by KRz-Yamaneko** [comment]
Hi folks,

Same question, I don't have the option to activate or not the GPU acceleration.

The tool is very cool, but I still have issue with my GPU acceleration.
My NVidia drivers are up-to-date for my 3080Ti, and I have the last cuda toolkit installed.

C:\Windows\System32>nvidia-smi
Sat Jan 13 23:44:56 2024
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 546.33                 Driver Version: 546.33       CUDA Version: 12.3     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                     TCC/WDDM  | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 3080 Ti   WDDM  | 00000000:0B:00.0  On |                  N/A |
|  0%   37C    P5              34W / 350W |   1020MiB / 12288MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+


C:\Windows\System32>nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2023 NVIDIA Corporation
Built on Wed_Nov_22_10:30:42_Pacific_Standard_Time_2023
Cuda compilation tools, release 12.3, V12.3.107
Build cuda_12.3.r12.3/compiler.33567101_0

**Comment by dossjjx** [comment]
Same issue here. Have an Nvidia GTX 1060 6GB GPU with latest Cuda installed. Windows 11.

**Reply by KRz-Yamaneko** [reply]
Solved with the nightly, the jan-win-x64-0.4.3-143.exe provide the GPU acceleration

**Reply by Sizzler** [reply]
Sounds awesome, gonna check it out, thanks for the info mate. 🙏

**Reply by dossjjx** [reply]
Confirming the nightly build fixed this
